gubernator/view_build.py

```diff
@@ -259,6 +259,8 @@ def get(self, prefix, job, build):
         if external_config is not None:
             if external_config.get('spyglass'):
                 spyglass_link = 'https://' + external_config['prow_url'] + '/view/gcs/' + build_dir
+                self.redirect(spyglass_link)
```

Comment:
If I knew for sure that nobody was using gubernator I would be fine with this.  There may be some internal users though?